### ISSUE TITLE
Terrible slowness when inspecting types or packages

### ISSUE THREAD

**Issue by edani** [ensime/ensime-server#717]
Inspecting types or packages with many elements (such as RichPresentationCompiler or org.ensime) now takes forever, and often fails due to the 30-second timeout.

The inferior-ensime-server log shows that the interval between `search:` messages (emitted by `locateSymbolPos`) is tens to hundreds of milliseconds. It used to be about 1ms, which leads me to believe the database is the problem, or perhaps logging, since the database generates tons of logs even though we discard them.

The problem appeared between commits c9b7cc6 and df64d7a68 My money is on d40a5d373 since it updates library versions.

**Comment by fommil** [comment]
looks like H2 picked up on the regression http://www.h2database.com/html/changelog.html

```
In version 1.3.183, indexes were not used if the table contains columns with a default value generated by a sequence. This includes tables with identity and auto-increment columns. This bug was introduced by supporting "rownum" in views and derived tables.
```

might be worth doing an upgrade and seeing if things are ok. Good to stay current with H2 when we can as they are always doing good work under the hood.

**Comment by edani** [comment]
This was fixed in #940
